apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -0,0 +1,228 @@
+/** @file Maker Discount Code Eligibility Checklist */
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import Button from "./Button";
+import SchoolAutocompleteDropdown from './SchoolAutocompleteDropdown';
+import SchoolNotFound from './SchoolNotFound';
+
+import SetupStep, {Status} from '../lib/kits/maker/ui/SetupStep';
+
+export default class EligibilityChecklist extends Component {
+  static propTypes = {
+    statusPD: PropTypes.oneOf(Object.values(Status)).isRequired,
+    statusStudentCount: PropTypes.oneOf(Object.values(Status)).isRequired,
+  };
+
+  state = {
+    statusYear: Status.UNKNOWN,
+    yearChoice: false, // stores the teaching-year choice until submitted
+    displayDiscountAmount: false,
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      country: 'United States',
+      hoc: '',
+      nces: '',
+      schoolName: '',
+      schoolCity: '',
+      schoolState: '',
+      schoolZip: '',
+      schoolType: '',
+      afterSchool: '',
+      tenHours: '',
+      twentyHours: '',
+      otherCS: false,
+      followUpFrequency: '',
+      followUpMore: '',
+      acceptedPledge: false
+    },
+    errors: {
+      invalidEmail: false
+    }
+  };
+
+  // Saves the teaching-year choice to trigger next step of actions
+  handleSubmit = () => {
+    this.setState({ statusYear: (this.state.yearChoice ? Status.SUCCEEDED : Status.FAILED )});
+  }
+
+  // Temporarily saves the teaching-year
+  // Param: accept - boolean - true for years to accept, false for deniable answers
```

Comment:
We have a specific syntax we use for these sorts of comments called JSDoc http://usejsdoc.org/about-getting-started.html. The documentation at that link isn't super great, and there are probably lots of places where we don't have our JSDoc quite right, but it would be good to try and match as closely as you can.

I also don't know that I really understand from the comment what "years to accept" or "deniable answers" mean. I'm also not sure I understand what `yearChoice` as a boolean means - might be we can come up with a different name for this.